app/services/import/persist-import.service.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+/**
+ * Creates or updates an imported licence and its child entities that have been transformed and validated
+ * @module PersistImportService
+ */
+
+const PersistImportService = require('./persist/persist-licence.service.js')
```

Comment:
If that is what we are naming the file we should name the value to match (and think this is the right way round!)

```suggestion
const PersistLicenceService = require('./persist/persist-licence.service.js')
```